com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs

```diff
@@ -40,7 +40,6 @@ unsafe struct ShaderVariablesDebugDisplay
         public int _DebugSingleShadowIndex;
 
         public int _DebugProbeVolumeMode;
-        public Vector3 _DebugDisplayPad0;
```

Comment:
Question: Why are we not catching issue currently with our test. We have debug mode test.
I would like to add a graphic test to catch this for the future, what kind of test should I add? thanks

Reply:
This is harmless, but takes up extra space in the constant buffer.  We need a generic mechanism to validate/enforce something like "std140" from Khronos that works across all APIs (and we don't have that at the moment).